.build/docker/docker-entrypoint.sh

```diff
@@ -1,53 +1,9 @@
-#!/bin/bash
+#!/bin/sh
 
-# Initialize flags for config.json and private key
-CONFIG_EXISTS=false
-PK_EXISTS=false
-
-# Check if config.json exists
-if [ -f /app/config.json ]; then
-  echo "config.json detected"
-  CONFIG_EXISTS=true
+if [ -f "/app/config.json" ] && [ -f "/app/data-proxy-private-key.json" ]; then
+  echo "Config and private key files found. Running with specific configuration."
+  exec ./dataproxy run --config /app/config.json --private-key-file /app/data-proxy-private-key.json "$@"
 else
-  echo "config.json not found"
+  echo "Config or private key file not found. Running with default configuration."
+  exec ./dataproxy run "$@"
```

Comment:
That's not entirely true, without the arguments it will look for `./config.json` and `./data-proxy-private-key.json` relative to the CWD of the process.